packager/media/event/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 Google LLC. All rights reserved.
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+add_library(media_event STATIC
+    combined_muxer_listener.cc
+    hls_notify_muxer_listener.cc
+    mpd_notify_muxer_listener.cc
+    multi_codec_muxer_listener.cc
+    muxer_listener_factory.cc
+    muxer_listener_internal.cc
+    vod_media_info_dump_muxer_listener.cc
+)
+
+target_link_libraries(media_event
+    file
+    mpd_media_info_proto
+    media_base
+    codecs
+)
+
+add_library(mock_muxer_listener STATIC
+    mock_muxer_listener.cc
+)
+
+target_link_libraries(mock_muxer_listener
+    gmock
+    media_event
+)
+
+add_executable(media_event_unittest
+    hls_notify_muxer_listener_unittest.cc
+    muxer_listener_internal_unittest.cc
+#    needs MockMpdNotifier from mpd/base
```

Comment:
Please mark this with something like `#  TODO(cmake): Re-enable when mpd/base is ported` so that we are more likely to find it again later.

Reply:
done